### ISSUE TITLE
[SQLReporting] Deploy-SSRSReport Function missing

### ISSUE THREAD

**Issue by buxel** [trondhindenes/PowershellModules#2]
Looks like the function "Deploy-SSRSReport" is missing in your [reporting module](https://github.com/trondhindenes/PowershellModules/blob/master/SQLReporting/SQLReporting.psm1). Do i need to import anything else before using it?

My error:

```
PS > Publish-SSRSReportsDirectory ".\SSRS_Files" "http://.../ReportService2010.asmx?wsdl"
VERBOSE: Deploy-SSRSReport -webServiceUrl http://.../ReportService2010.asmx?wsdl -rdlFile
[...ReportFile...].rdl -reportFolder  -force -Credentials
The term 'Deploy-SSRSReport' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At [...]\SqlReporting\SqlReporting.psm1:230 char:34
+                 Deploy-SSRSReport <<<<  -webServiceUrl $webServiceUrl -rdlFile $ReportFullname -reportFolder $TargetFolder -force $true -Credentials $Credentials
    + CategoryInfo          : ObjectNotFound: (Deploy-SSRSReport:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by buxel** [comment]
It took me loger to file this issue than fixing the problem >.<
Anyways, check my [PR](https://github.com/trondhindenes/PowershellModules/pull/3)

**Comment by trondhindenes** [comment]
resolved thru PR #3
